lib/internal/Magento/Framework/View/Page/Config.php

```diff
@@ -539,6 +544,31 @@ public function addBodyClass($className)
         return $this;
     }
 
+    /**
+     * Retrieve the additional meta assets
+     *
+     * @return array
+     */
+    public function getAdditionalMetaAssets()
```

Comment:
@vasilii-b @larsroettig adding public methods to an @api class is a backward incompatible change that cannot be accepted to patch version.

Can the code be refactored to avoid that?

i.e. :

- introducing private implementation to Config Renderer
- introducing a new class that provides the required public methods

Reply:
@sivaschenko sorry I don't see the @api annotations there. Introducing a new class should be possible solutions